BUILD.bazel

```diff
@@ -20,117 +20,126 @@ exports_files([
-    ],
-)
-
-cc_library(
-    name = "experimental-artifactregistry",
-    deps = [
-        "//google/cloud/artifactregistry:google_cloud_cpp_artifactregistry",
-    ],
-)
-
-cc_library(
-    name = "experimental-artifactregistry_mocks",
-    deps = [
-        "//google/cloud/artifactregistry:google_cloud_cpp_artifactregistry_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-asset",
-    deps = [
-        "//google/cloud/asset:google_cloud_cpp_asset",
-    ],
-)
-
-cc_library(
-    name = "experimental-asset_mocks",
-    deps = [
-        "//google/cloud/asset:google_cloud_cpp_asset_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-assuredworkloads",
+EXPERIMENTAL_LIBRARIES = [
+    "accessapproval",
+    "accesscontextmanager",
+    "apigateway",
+    "appengine",
+    "artifactregistry",
+    "asset",
+    "assuredworkloads",
+    "automl",
+    "billing",
+    "binaryauthorization",
+    "channel",
+    "cloudbuild",
+    "composer",
+    "container",
+    "containeranalysis",
+    "datacatalog",
+    "datamigration",
+    "debugger",
+    "dlp",
+    "eventarc",
+    "filestore",
+    "functions",
+    "gameservices",
+    "gkehub",
+    "iap",
+    "ids",
+    "iot",
+    "kms",
+    "logging",
+    "managedidentities",
+    "memcache",
+    "monitoring",
+    "networkmanagement",
+    "notebooks",
+    "orgpolicy",
+    "osconfig",
+    "oslogin",
+    "policytroubleshooter",
+    "privateca",
+    "pubsublite",
+    "recommender",
+    "redis",
+    "resourcemanager",
+    "retail",
+    "scheduler",
+    "securitycenter",
+    "servicecontrol",
+    "servicedirectory",
+    "servicemanagement",
+    "serviceusage",
+    "shell",
+    "storagetransfer",
+    "talent",
+    "texttospeech",
+    "tpu",
+    "trace",
+    "translate",
+    "videointelligence",
+    "vision",
+    "vmmigration",
+    "vpcaccess",
+    "webrisk",
+    "websecurityscanner",
+    "workflows",
+]
+
+TRANSITION_LIBRARIES = [
+    "secretmanager",
+    "tasks",
+]
+
+GA_LIBRARIES = [
+]
+
+[cc_library(
+    name = "experimental-{library}".format(library = library),
+    deps = [
+        "//google/cloud/{library}:google_cloud_cpp_{library}".format(library = library),
+    ],
+) for library in EXPERIMENTAL_LIBRARIES]
+
+[cc_library(
+    name = "experimental-{library}_mocks".format(library = library),
+    deps = [
+        "//google/cloud/{library}:google_cloud_cpp_{library}_mocks".format(library = library),
+    ],
+) for library in EXPERIMENTAL_LIBRARIES]
+
+[cc_library(
+    name = "experimental-{library}".format(library = library),
+    deprecation = "this library is now GA, please use //:{library} instead.".format(library = library),
+    tags = ["manual"],
     deps = [
-        "//google/cloud/assuredworkloads:google_cloud_cpp_assuredworkloads",
+        "//google/cloud/{library}:google_cloud_cpp_{library}".format(library = library),
     ],
-)
+) for library in TRANSITION_LIBRARIES]
```

Comment:
same option

Reply:
Ditto, but if you know of a Python / Starlark trick to get these working I would be glad to know.